galley/pkg/source/fs/fssource.go

```diff
@@ -295,22 +323,138 @@ func (s *fsSource) Start() (chan resource.Event, error) {
 			}
 		}
 	}()
-	return s.ch, nil
+	return nil
 }
 
 // New returns a File System implementation of runtime.Source.
 func New(root string, schema *schema.Instance, config *converter.Config) (runtime.Source, error) {
 	fs := &fsSource{
-		config:          config,
-		root:            root,
-		kinds:           map[string]bool{},
-		fileResorceKeys: map[string][]*fileResourceKey{},
-		donec:           make(chan struct{}),
-		shas:            map[resource.FullName][sha1.Size]byte{},
-		version:         0,
+		config:           config,
+		root:             root,
+		kinds:            map[string]bool{},
+		fileResourceKeys: map[string][]*fileResourceKey{},
+		donec:            make(chan struct{}),
+		shas:             map[resource.FullName][sha1.Size]byte{},
+		version:          0,
 	}
 	for _, spec := range schema.All() {
 		fs.kinds[spec.Kind] = true
 	}
 	return fs, nil
 }
+
+type fileResource struct {
```

Comment:
Yeah ... I renamed it to `fileResource`, but still a copy & paste. I moved to this file since many of the functions became members of the source struct.